### ISSUE TITLE
Usage example not working

### ISSUE THREAD

**Issue by danieljuschus** [edsaac/streamlit-PyVista-viewer#8]
The [usage example](https://github.com/edsaac/streamlit-PyVista-viewer#usage-example) is not currently working for me. No errors, but the 3D model is not shown.

There is also a deprecation warning: `PyVistaDeprecationWarning: pythreejs backend is deprecated and is planned for future removal.`

The examples on the page on the Streamlit Community Cloud don't work either anymore: https://stpyvista.streamlit.app/,

Windows 10, Python 3.11, pyvista 0.38.5, stpyvista 0.0.5, streamlit 1.20.0

**Comment by edsaac** [comment]
Indeed, pyvista's `pythreejs` backend is getting deprecated in favor of [trame](https://kitware.github.io/trame/index.html) . I was moving to the `panel` backend, which is arguably better than pythreejs, but that one is also deprecated now so never merged that branch. :/  I'm looking into integrating trame

**Comment by edsaac** [comment]
Sorry, didn't mean to close the issue. I'll try to port `stpyvista` to `trame` eventually in a next version. For now, fixing `pyvista` to a previous version (maybe 0.36.0) should do the trick, but cannot confirm that yet.

**Comment by anciaux** [comment]
I cannot find any combination fixing the pyvista version to something older that fixes the issue.  Could someone do it ? 

is so could I get the version of all packages ?

Also, the trame is interesting when do plan to have this included ? Do you need help somehow ?

**Comment by edsaac** [comment]
I tried many version combinations and could not figure out the problem. My guess now is that something is broken in the `threejs` library and I gave up. So I released version 0.0.6 of `stpyvista` to just use the `panel` backend, which --although deprecate-- still works! 

You can upgrade to version 0.0.6 via pip (`pip install stpyvista -U`). Tests seem to be passing [in st.pyvista.streamlit.app](https://stpyvista.streamlit.app/) but lmk if something else broke.

**Comment by anciaux** [comment]
Thanks you so much. It indeed worked!

**Comment by danieljuschus** [comment]
Not working for me unfortunately, I keep getting errors like 

```  
File "C:\Users\jusc_da\Documents\AirFuCS\airfucs-calc\venv\Lib\site-packages\panel\io\save.py", line 295, in save
    with io.open(filename, mode="w", encoding="utf-8") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\______\\AppData\\Local\\Temp\\tmpz13yuo84.html'
```

**Comment by edsaac** [comment]
That looks like a permissions configuration issue with your computer, when `panel` attempts to create temporal files.

**Comment by jeac1771** [comment]
Hello, I have the same problem, how did you solve it?: PermissionError: [Errno 13] Permission denied:.

Thanks you

**Comment by danieljuschus** [comment]
I have not yet found a solution. Unfortunately I am currently restricted to a Windows machine without admin rights. Are you in the same situation perhaps?

**Comment by jaedong27** [comment]
I think that this problem results from accessing temp file in "root/panel/io/save.py", line 295 when NamedTemporaryFile in "root/__init__.py", line 84 hold the temp file.
To handle this problem, I changed "root/pannel/io/save.py", line 83 ~ 86
```python
        # Create HTML file
        with NamedTemporaryFile(mode='a+', suffix='.html') as model_html:
            geo_pan_pv.save(model_html.name, resources=INLINE)
            panel_html = model_html.read()
```
To
```python
        # Create HTML file
        temp_path = ""
        with NamedTemporaryFile(mode='a+', suffix='.html') as model_html:
            temp_path = model_html.name
        geo_pan_pv.save(temp_path, resources=INLINE)
        with open(temp_path, 'r', encoding="utf-8") as model_html:
            panel_html = model_html.read()
```
This code just get temp file name from NamedTemporaryFile and release that temp file.
Then it process other things.

**Comment by edsaac** [comment]
I got rid of the whole temporary file in version 0.0.7 of `stpyvista`. Test seem to be passing at [https://stpyvista.streamlit.app/](https://stpyvista.streamlit.app/), so you should be good after upgrading (`pip install stpyvista -U`).  But again, lmk if something failed!

**Comment by jaedong27** [comment]
I checked working about version 0.0.7. Thank you!

**Comment by danieljuschus** [comment]
Works for me too, thanks! Shall we open a new issue for trame?

**Comment by danieljuschus** [comment]
Is it just me or is the translation of the view using the middle mouse button not working anymore with the panel backend?
